src/std_picture.h

```diff
@@ -33,11 +33,14 @@
 /*!
  * \brief RGB pixel color
  */
+// uses floats right now but should be converted into a packed
+// 32-bit integer for efficient parameter passing;
+// see the wip render queue branch
 struct PGEColor
 {
-    uint8_t r = 0;
-    uint8_t g = 0;
-    uint8_t b = 0;
+    float r = 0;
+    float g = 0;
+    float b = 0;
```

Comment:
Hmm, let's talk more about this soon. I think it's possible that some incompatible code mixing took place when I initially rebased the feature on the current main branch, because I remember my initial code mixing `float`s and `uint8_t`s. After this merge, I can make a mini PR that converts all colors into `uint32_t`-backed classes as I discussed in the previous comment.